ui/components/multichain/asset-picker-amount/asset-picker-amount.tsx

```diff
@@ -55,21 +57,36 @@ export const AssetPickerAmount = ({
   const selectedAccount = useSelector(getSelectedInternalAccount);
   const t = useI18nContext();
 
-  const isFiatPrimary = useSelector(getIsFiatPrimary);
-
   const { swapQuotesError } = useSelector(getCurrentDraftTransaction);
   const isDisabled = !onAmountChange;
   const isSwapsErrorShown = isDisabled && swapQuotesError;
 
   const [isFocused, setIsFocused] = useState(false);
+  const [isNFTInputChanged, setIsTokenInputChanged] = useState(false);
+
+  const handleChange = useCallback(
+    (newAmountRaw, newAmountFormatted) => {
+      if (!isNFTInputChanged && asset.type === AssetType.NFT) {
```

Comment:
How come the same error handling is not done for tokens?

Reply:
We allow for 0 amount ERC20 transfers but not NFTs – not sure though